### PR TITLE
Translation of help/index/intro/setup and setup in french

### PR DESCRIPTION
**Pull request by justicengom**
I add a new file (help.fr.md) in french https://github.com/justicengom/training/edit/master/docs/help.md

### CODE REVIEW

File: docs/basic_training/index.fr.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+description: Aperçu du matériel de formation Nextflow de base
+hide:
+    - toc
+---
+
+## Bienvenue
+
+## Bienvenue à l'atelier de formation Nextflow
+
+Nous sommes heureux de vous accompagner sur le chemin de redaction de workflows scientifiques reproductibles et évolutifs en utilisant Nextflow. Ce guide complète toute la documentation de Nextflow - si vous avez des doutes, consultez la documentation située [ici](https://www.nextflow.io/docs/latest).
```
Comment (maxulysse) [inline docs/basic_training/index.fr.md:11]:
```suggestion
Nous sommes heureux de vous accompagner sur le chemin de rédaction de workflows scientifiques reproductibles et évolutifs en utilisant Nextflow. Ce guide complète toute la documentation de Nextflow - si vous avez des doutes, consultez la documentation située [ici](https://www.nextflow.io/docs/latest).
```

File: docs/basic_training/index.fr.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+À la fin de ce cours, vous devriez:
+
+1. Maîtriser la redaction de workflows Nextflow
+2. Connaître les concepts de base de Nextflow : canaux, processus et opérateurs.
```
Comment (maxulysse) [inline docs/basic_training/index.fr.md:16]: @mribeirodantas Would you translate `channel` -> `canaux` and so on?

File: docs/basic_training/setup.fr.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+-   Git
+-   [Docker](https://docs.docker.com/get-docker/)
+
+#### Exigences optionnelles pour ce tutoriel
+
+-   [Singularity](https://github.com/sylabs/singularity) 2.5.x (or later)
+-   [Conda](https://conda.io/) 4.5 (or later)
+-   [Graphviz](http://www.graphviz.org/)
+-   [AWS CLI](https://aws.amazon.com/cli/)
+-   A configured AWS Batch computing environment
```
Comment (maxulysse) [inline docs/basic_training/setup.fr.md:32]:
```suggestion
-   [Singularity](https://github.com/sylabs/singularity) 2.5.x (ou plus)
-   [Conda](https://conda.io/) 4.5 (ou plus)
-   [Graphviz](http://www.graphviz.org/)
-   [AWS CLI](https://aws.amazon.com/cli/)
-   Un environnement AWS Batch configuré
```

File: docs/basic_training/setup.fr.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+### Télécharger Nextflow
+
+Entrez cette commande dans votre terminal :
+
+```bash
+wget -qO- https://get.nextflow.io | bash
+```
+
+Or, if you prefer `curl`:
```
Comment (maxulysse) [inline docs/basic_training/setup.fr.md:42]:
```suggestion
Ou, vous préférez `curl`:
```

File: docs/basic_training/setup.fr.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+System: Linux 5.15.0-47-generic
+Runtime: Groovy 3.0.13 on OpenJDK 64-Bit Server VM 17.0.3-internal+0-adhoc..src
+Encoding: UTF-8 (UTF-8)
+```
+
+### Resources de Gitpod 
+
+-   Gitpod vous offre 500 crédits gratuits par mois, ce qui équivaut à 50 heures d'utilisation gratuite de l'environnement en utilisant l'espace de travail standard (jusqu'à 4 cœurs, 8 Go de RAM et 30 Go d'espace de stockage).
+-   Il existe également une option de grand espace de travail qui vous permet d'utiliser jusqu'à 8 cœurs, 16 Go de RAM et 50 Go d'espace de stockage. Cependant, le grand espace de travail utilisera vos crédits gratuits plus rapidement et vous aurez moins d'heures d'accès à cet espace.
+-   Gitpod s'arrête au bout de 30 minutes d'inactivité et conserve les modifications jusqu'à deux semaines (voir la section suivante pour reouvrir une session interrompue).
```
Comment (maxulysse) [inline docs/basic_training/setup.fr.md:132]:
```suggestion
-   Gitpod s'arrête au bout de 30 minutes d'inactivité et conserve les modifications jusqu'à deux semaines (voir la section suivante pour réouvrir une session interrompue).
```

**Comment by mribeirodantas** [comment]
You had already translated the `index.fr.md` and `mkdocs.yml` files in another Pull Request, and now we have a conflict. If you need help resolving this, please let me know 😃

**Comment by justicengom** [comment]
> You had already translated the `index.fr.md` and `mkdocs.yml` files in another Pull Request, and now we have a conflict. If you need help resolving this, please let me know 😃

It is two differents files from different folders :
https://github.com/justicengom/training/blob/master/docs/index.fr.md
https://github.com/justicengom/training/blob/master/docs/basic_training/index.fr.md

**Comment by mribeirodantas** [comment]
Click on `Files changed` here on the GitHub interface. You'll see your PR contains changes in both files. I think that you created a new branch to work on before pulling the updated master 🤔

**Comment by mribeirodantas** [comment]
Hello, @justicengom. Be careful, I committed twice to fix a few things in this Pull Request. So before committing again in your local machine, make sure you pull (`git pull`) and have an up-to-date workspace to work on 😃

**Comment by justicengom** [comment]
> Hello, @justicengom. Be careful, I committed three times to fix a few things in this Pull Request. So before committing again in your local machine, make sure you pull (`git pull`) and have an up-to-date workspace to work on 😃

Well received and thank you very much @mribeirodantas

**Comment by mribeirodantas** [comment]
Thanks for the amazing work, @justicengom! I hope you're still interested in translating the rest of the training material 🤩. As soon as @maxulysse approves the translation, I will merge it 🥳

**Comment by justicengom** [comment]
> Thanks for the amazing work, @justicengom! I hope you're still interested in translating the rest of the training material 🤩. As soon as @maxulysse approves the translation, I will merge it 🥳

Thank you @mribeirodantas  Yes sure, i will continue

File: docs/basic_training/cache_and_resume.fr.md [hunk 6]
```diff
@@ -0,0 +1,327 @@
...
+
+    Sauvegardez l'extrait ci-dessus dans un fichier nommé `template.html`. Lancez ensuite cette commande (en utilisant l'identifiant correct pour votre exécution, par exemple `tiny_fermat`) :
+
+    ```bash
+    nextflow log tiny_fermat -t template.html > prov.html
+    ```
+
+    Enfin, ouvrez le fichier `prov.html` avec un navigateur.
+
+## reprise du dépannage
```
Comment (maxulysse) [inline docs/basic_training/cache_and_resume.fr.md:201]:
```suggestion
## Dépanner la reprise
```

**Review by maxulysse — COMMENTED** [review]
Fantastique!!!

**Comment by maxulysse** [comment]
@justicengom un seul changement nécessaire à mon avis.
Merci pour ce haut niveau de traduction.

Je pense qu'on pourra regarder plus tard l'ensemble du tutoriel et voir si tout est bien cohérent.
(Je pense à l'utilisation de `déboggage` et `dépannage` par exemple)
Mais bravo !!!

**Comment by mribeirodantas** [comment]
A lot of work has been put in this PR and I think it's time for it to get merged. Thanks, @justicengom and @maxulysse for all the amazing work. We can come back to it in the near future to improve it if you feel it's required.

I'm so excited about having some sections translated into French 🤩

**Review by mribeirodantas — DISMISSED** [review]

**Comment by abhi18av** [comment]
Aaad, @justicengom is back with another PR https://github.com/nextflow-io/training/pull/239 👏

**Pull request merged**